src/ActionList/ActionList.examples.stories.tsx

```diff
@@ -187,10 +189,20 @@ export function AsyncListWithSpinner(): JSX.Element {
         filter. This pattern can be found in branch selection menus via the SelectPanel component.
       </p>
 
-      <TextInput onChange={filter} placeholder="Search branches" sx={{m: 2, mb: 0, width: 'calc(100% - 16px)'}} />
-      {results.length === 0 ? (
-        <Text sx={{display: 'block', fontSize: 1, m: 2}}>No branches match that query</Text>
-      ) : null}
+      <FormControl sx={{m: 2, mb: 0, width: 'calc(100% - 16px)'}}>
+        <FormControl.Label>Search branches</FormControl.Label>
+        <TextInput onChange={filter} block />
+      </FormControl>
+      <VisuallyHidden
+        isVisible={results.length === 0}
+        aria-live="polite"
+        aria-atomic="true"
+        as={Text}
+        sx={{display: 'block', fontSize: 1, m: 2}}
+      >
+        {results.length === 0 ? 'No branches match that query' : `${results.length} branches match that query`}
+      </VisuallyHidden>
```

Comment:
For live region support, we have an internal `LiveRegion` component that we could try to consolidate on if it's helpful. 

Right now it's not a standalone component (meaning it needs two components) but we could totally add a standalone variant to it that's just like `<LiveRegion>The message</LiveRegion>` if it would be helpful for any live region based a11y fixes 👀

Reply:
Thanks for reminding me we had this! I just had to update `LiveRegionOutlet` so we could visually show the `No branches match that query` message.